src/main/java/org/elasticsearch/app/Harvester.java

```diff
@@ -562,6 +559,11 @@ public Harvester rdfIndexType(String indexType) {
         return this;
     }
 
+    public Harvester setIncrementally(boolean incrementally) {
```

Comment:
I saw quite many very similar functions here, repeating sam pattern:
rdfNormalizationObj
rdfNormalizationMissing
rdfNormalizationProp

I would think of [Lombok](https://projectlombok.org/features/GetterSetter) to do this ... and set up getters with props like:
lombok.accessors.chain
lombok.accessors.fluent

Reply:
Other similar methods are from someone else from the forked repo so I used the same pattern. I am planning a big refactor so I will add this to the list.